lib/jellyfish/room.ex

```diff
@@ -47,9 +47,22 @@ defmodule Jellyfish.Room do
     GenServer.start_link(__MODULE__, args)
   end
 
-  @spec get_state(pid()) :: t()
+  def start(args) do
+    GenServer.start(__MODULE__, args)
+  end
+
+  @spec get_state(pid()) :: t() | nil
   def get_state(room_pid) do
-    GenServer.call(room_pid, :state)
+    try do
+      GenServer.call(room_pid, :state)
+    catch
+      :exit, {:noproc, {GenServer, :call, [^room_pid, :state, _timeout]}} ->
+        Logger.warn(
+          "During get state of room #{inspect(room_pid)}, process exited because this process doesn't exist"
```

Comment:
> process exited because this process doesn't exist

Umm, that's quite confusing :stuck_out_tongue: 
How about:
```suggestion
          "Cannot get state of #{inspect(room_pid)}, the room's process doesn't exist anymore"
```